### ISSUE TITLE
Collapse code on small screens

### ISSUE THREAD

**Issue by RangerMauve** [baku89/pentool#2]
Hi! I really want to try this out on my mobile devices with pen support to see how it feels, but the view gets messed up because it's trying to fit the code along side the drawing area. It'd be nice if the code view could be toggled so that you could have more space for writing, and to have it auto-hide on small screens.

**Comment by baku89** [comment]
Thank you for your suggestion!

I'm also thinking to let the app work on mobile devices -- I came to wonder it would be great if I could use it with iPad Pro.
I have to focus on my paid works so far, yet I'd really like to support iOS.

Thanks for the feedback!

**Comment by RangerMauve** [comment]
Yeah I get whatcha mean. This seems like it'd be ideal for an iPad pro and the such.

I might have some time to play around with the source eventually and see if I can get it to work. 😁
